sstmac/common/event_scheduler.h

```diff
@@ -528,21 +526,29 @@ class Component : public ComponentParent
 };
 
 
+namespace detail {
+  // Class to provoide the same virtual interface as SST::Subcomponent, mainly
+  // to silence warnings about override, but also because we don't change the
+  // interface when Core is unavalible. 
+  class CoreSubComponentStub {
+    public:
```

Comment:
Where does this get used now? It might just be GitHub not showing this clearly to me.

Reply:
Apparently this snuck back in some how 🤷‍♀️

Reply:
Gone